packages/by-name/kata/kata-runtime/package.nix

```diff
@@ -144,5 +147,10 @@ buildGoModule rec {
     in
     [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
 
+  passthru.cmdline = {
```

Comment:
Why is the cmdline attribute introduced in c07f3782ef78cde7be99346fde85ce0d9ef7a55b, which disables that Kata check?